frontend/src/components/TopAppBar.tsx

```diff
@@ -24,6 +24,7 @@ const TopAppBar = (props: { children: React.ReactNode }) => {
         sx={{
           width: '100vw',
           height: 'calc(100vh - 64px)',
+          display: 'inline',
```

Comment:
To ensure that only the Chats page uses display inline, in TopAppBar you can create an optional prop called `display` which defaults to 'flex'. Then, only in your Chats page, you can pass a `display: 'inline'` prop.